phpstan.neon.dist

```diff
@@ -12,4 +12,8 @@ parameters:
         - %currentWorkingDirectory%/tests
     excludePaths:
         - %currentWorkingDirectory%/src/Annotation/ReadOnly.php
-     
+        - %currentWorkingDirectory%/tests/Fixtures/TypedProperties/UnionTypedProperties.php
+        - %currentWorkingDirectory%/tests/Fixtures/TypedProperties/UnionTypedProperties.php
+        - %currentWorkingDirectory%/tests/Metadata/Driver/UnionTypedPropertiesDriverTest.php
+        - %currentWorkingDirectory%/tests/Serializer/BaseSerializationTest.php
```

Comment:
I will check tomorrow if we can setup differently - it would be better to do not ignore test files.

Reply:
I didn't find any solution to exclude it. May we can split the tests for PHP8.0 only into separate class and ignore them for PHP7.4 static analytics? Configuration for the jobs can be found here: `.github/workflows/static-analysis.yaml`.